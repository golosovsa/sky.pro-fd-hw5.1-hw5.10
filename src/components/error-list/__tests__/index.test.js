import ShallowRenderer from 'react-test-renderer/shallow'
import ErrorList from '..'
import ErrorItem from '../../error-item'

describe('<ErrorList />', () => {
  describe('Snapshot test', () => {
    it('should render successfully and match snapshot', () => {
      const renderer = new ShallowRenderer()
      const tree = renderer.render(
        <ErrorList>
          {[1, 2, 3].map((item) => (
            <ErrorItem key={item} error={item} message={item} />
          ))}
        </ErrorList>
      )

      expect(tree).toMatchSnapshot()
    })
  })
})
