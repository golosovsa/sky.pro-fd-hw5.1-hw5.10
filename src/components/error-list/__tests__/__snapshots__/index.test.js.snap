// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorList /> Snapshot test should render successfully and match snapshot 1`] = `
<styled.div>
  <ErrorItem
    error={1}
    message={1}
  />
  <ErrorItem
    error={2}
    message={2}
  />
  <ErrorItem
    error={3}
    message={3}
  />
</styled.div>
`;
